### ISSUE TITLE
Implement automatic parameter loading for all the processes 

### ISSUE THREAD

**Issue by nkx111** [rest-for-physics/framework#42]
It is time to implement automatic parameter loading from rml text to class data member. We shall verify each process that

1. All the data members are named following our naming convention
2. The method `InitFromConfigFile()` has only simple logic of parameter reading: `fXXX=StringToDouble(GetParameter("xXX","10"))`

**Comment by jgalan** [comment]
Will constructions such as this one be properly interpreted by default?

```
std::vector <double> fTimeWindow;
fTimeWindow = StringToElements(GetParameter("timeWindow", "-1,-1"), ",");
```

This construction appears for example at: https://github.com/rest-for-physics/rawlib/blob/master/src/TRestRawVetoAnalysisProcess.cxx

**Comment by nkx111** [comment]
Yes, it could. But somehow the string to vector style is different. The string form should be like: `{1,2,3,4,5}`.

```
std::vector<double> vec;
any(vec).ParseString("{1,2,3,0.5}"); // vec will become {1,2,3,0.5}
```

It is implemented in https://github.com/rest-for-physics/framework/blob/f83c73dae97f781827e55edbf8e3cf0336a360ac/source/framework/core/src/startup.cpp#L181
